app/src/main/java/com/cupcake/jobsfinder/domain/model/JobWithJobTitle.kt

```diff
@@ -1,13 +1,13 @@
 package com.cupcake.jobsfinder.domain.model
 
-data class Job(
+data class JobWithJobTitle(
```

Comment:
i see last name is perfect, it is the name class no need to say with job title because job is already have job title.